controls/radfiledialogs/radopenfiledialog.md

```diff
@@ -83,6 +83,16 @@ You can get only the name of the selected files, without the full path, via the
 
 The __FileName__ property can be set manually. This will change the value displayed in the selected file autocomplete box area. Note that setting this won't change the selected item in the list with the files.
 
+## Saving the last used directory
+
+You can save the last used directory by setting the __RestoreDirectory__ property of the RadOpenFileDialog. After setting this property to __True__ and opening a file the __InitialDirectory__ of this RadOpenFileDialog instance will be the parent folder of the opened file.
```

Comment:
Maybe we can use "will be set to the parent folder" rather than "will be the parent folder".